open-authoring/OpenAuthoringModalContainer.tsx

```diff
@@ -28,6 +32,17 @@ export const OpenAuthoringModalContainer = ({ error }: Props) => {
     setAuthPopupDisplayed(false)
   }
 
+  useEffect(() => {
+    ;(async () => {
+      if (uninterpretatedError) {
+        const contextualError = await interpretError(uninterpretatedError)
```

Comment:
The variables names of interpretError, uninterpretatedError are a bit ambiguous what they actually represent. 
I think it might read better if it was like: getOpenAuthErrorDescriptor(openAuthError)